### PR TITLE
fix(GlobalSettings): Add VR panel

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

**Comment by jourdain** [comment]
@floryst @martinken

File: src/components/core/GlobalSettings/template.html [hunk 1]
```diff
@@ -85,4 +85,57 @@
...
+        </v-flex>
+      </v-layout>
+    </div>
+    <v-divider :class="$style.divider" />
+    <v-layout row align-center :class="$style.lineTopPadding" wrap>
+      <v-flex xs5>
+        <span class="body-2">Physical scale</span>
+      </v-flex>
+      <v-flex xs7>
+        <v-slider
```
Comment (floryst) [inline src/components/core/GlobalSettings/template.html]: Maybe add a textbox to show the current value of the physical scale slider?

**Comment by floryst** [comment]
I briefly tried this on my phone in a cardboard, and the camera looks like it's viewing from inside the data? Not sure if I'm using it correctly, but this is just what I've experienced. Also, for some reason the molecular bonds in the molecule example disappear from my view.

**Comment by jourdain** [comment]
I'm focusing on the Vive. I'm not expecting to work on mobile that well

File: src/components/core/GlobalSettings/template.html (hunk 1 above)
Comment (jourdain) [reply src/components/core/GlobalSettings/template.html]: It's more a zoom slider, having a value does not make sense.

**Comment by jourdain** [comment]
I'm somewhat ready... Meaning everything that I want is working. We can always make small fix if need be, especially if Ken find the issue for the VR part regarding the physical orientation.

**Comment by jourdain** [comment]
The previous comment was for you @floryst (sorry)

**Pull request merged**